dnf-behave-tests/fixtures/specs/needs-restarting-updates/abcde-2.9.3-1.fc29.spec

```diff
@@ -0,0 +1,22 @@
+Name:           abcde
+Version:        2.9.3
```

Comment:
I prefer to use simple versions for the test packages, like `1.0`, so that you don't need to squint when you have multiple versions of a package like 3.14.1.1 and 3.14.7.1 and try to figure out which is which :sunglasses: But feel free to keep it here, not much of an issue in this case.